packages/realm-app-importer/src/cli.ts

```diff
@@ -92,6 +92,36 @@ function resolveAppTemplates(paths: string[]): Record<string, string> {
   );
 }
 
+const DEFAULTS = {
+  baseUrl: process.env.REALM_BASE_URL || "http://localhost:9090",
+  username: process.env.REALM_USERNAME || "unique_user@domain.com",
+  password: process.env.REALM_PASSWORD || "password",
+  publicKey: process.env.REALM_PUBLIC_KEY,
+  privateKey: process.env.REALM_PRIVATE_KEY,
+};
+
+type CredentialsOptions = {
+  username: string;
+  password: string;
+  publicKey: string | undefined;
```

Comment:
Any reason to not use `publicKey?: string` ?

Reply:
Basically, `username` and `password` has default values and the `publicKey` and `privateKey` doesn't.